### ISSUE TITLE
FreshIOC.Container.Register<HttpClient>() no constructor takes 1 argument

### ISSUE THREAD

**Issue by allenmoatallen** [rid00z/FreshMvvm#60]
In the newest version did this change? I had that line in previous versions without any complaints but now it won't compile.

I have switched it to `FreshIOC.Container.Register(typeof(HttpClient));` but not sure that is the proper way.

**Comment by WiteCastle** [comment]
Yeah, I believe it's now `Register<InterfaceClass,ImplementationClass>()`. So this should probably be how you want to do it now:

`FreshIOC.Container.Register<HttpClient, HttpClient>();`
